fonts/mscore/glyphnames.json

```diff
@@ -745,5 +743,81 @@
     },
     "noteLongaSquareUp": {
         "codepoint": "U+e1c4"
+    },
```

Comment:
I'm just starting, so maybe I miss something but if the font doesn't have these glyphs, then they shouldn't be listed here.